tools/halide_image.h

```diff
@@ -184,6 +184,10 @@ class Image {
         return &(contents->buf);
     }
 
+    uint8_t *alloc() const {
+        return contents->alloc;
```

Comment:
I have updated the patch per the review comments. Let me know if there are any further changes you would like me to make.

 

Thanks,

Dan

—
Reply to this email directly or view it on GitHub <https://github.com/halide/Halide/pull/1037/files#r53413448> .  <https://github.com/notifications/beacon/APZNJLKbO78TwbNQkXwVKbzY1IPLvbXlks5plm04gaJpZM4HclUO.gif>